### DISCUSSION TITLE
How to translate the application?

### DISCUSSION THREAD

**Discussion by GET100PERCENT** [sheikhhaziq/gyawun_music#35] [General]
Bro, i like this app very much.
I like to translate the application into Odia language.
Explain me how to translate.

**Comment by sheikhhaziq** [comment]
In order to add new language:

1. you need to fork this repo.
2. Go to folder lib/l10n.
3. Create a new file intl_xx.arb.
4. In place of `xx`, write two digit language code.
5. Copy content of  [This](https://github.com/sheikhhaziq/vibemusic/blob/main/lib/l10n/intl_en.arb) file.
6. Change it accordingly.
7. Make a pull request.

**Comment by GET100PERCENT** [comment]
The translation for "play next" is not added in [this](https://github.com/sheikhhaziq/vibemusic/blob/main/lib/l10n/intl_en.arb)  file . So what I have to do now to add it's odia translation?

**Reply by sheikhhaziq** [reply]
Add `"Play_next":"translation goes here"` at the end of file

**Comment by GET100PERCENT** [comment]
How to play the songs having video only version on YouTube, in the search results they are not available?

**Reply by sheikhhaziq** [reply]
We are using youtube music api, not youtube api. That's why you can only search for actual songs on youtube music not youtube.

**Reply by GET100PERCENT** [reply]
Can't it be possible to use YouTube API along with YouTube Music API, so that audio stream of the video only songs can also be played?
It has been implemented in other similar Foss clients of YouTube Music, including Vi Music, Innertune, Musify and also the official YouTube Music app gives this feature.

**Reply by sheikhhaziq** [reply]
Sure, I will.

**Reply by sheikhhaziq** [reply]
Video search tab has been added to search section in new beta release, you can check it out now.

**Reply by GET100PERCENT** [reply]
Thanks bro

**Comment by GET100PERCENT** [comment]
Is it possible to have installed both the normal version and the beta version at the same time?

**Reply by sheikhhaziq** [reply]
Not yet, but I can make that happen.

**Reply by GET100PERCENT** [reply]
Then please make that happen, it will also be useful in testing the beta versions, while not uninstalling the stable version.

**Reply by sheikhhaziq** [reply]
Sure

**Reply by sheikhhaziq** [reply]
With new beta release, you can now keep botha beta and stable releases in your device.

**Reply by GET100PERCENT** [reply]
Thanks 👍
